### PR TITLE
add websocket support info on subnet

### PR DESCRIPTION
**Pull request by yulin-dong**

### CODE REVIEW

File: docs/build/tutorials/platform/subnets/setup-dfk-node.md [hunk 1]
```diff
@@ -104,3 +104,9 @@ cd subnet-evm
 ./scripts/build.sh $GOPATH/src/github.com/ava-labs/avalanchego/build/plugins/mDV3QWRXfwgKUWb9sggkv4vQxAQR4y2CyKrt5pLZ5SzQ7EHBv
 ./build/avalanchego --whitelisted-subnets Vn3aX6hNRstj5VHHm63TCgPNaeGnRSqCYXQqemSqDd2TQH4qJ
 ```
+
+## Appendices
+
+### Websocket Support on DFK Subnet
```
Comment (patrick-ogrady) [inline docs/build/tutorials/platform/subnets/setup-dfk-node.md:110]: I think this should be placed next to the other API metadata for DFK? I don't think the end of this tutorial is the right place unless we include all this info: 
image

File: docs/build/tutorials/platform/subnets/wagmi.md [hunk 2]
```diff
@@ -32,3 +32,7 @@ We created a basic WAGMI Explorer [https://trywagmi.xyz](https://trywagmi.xyz) t
 * Chain ID: 11111
 * Symbol: WGM
 * Explorer: https://subnets.avax.network/wagmi/wagmi-chain-testnet/explorer
+
+## WebSocket Support
+
+* URL: wss://subnets.avax.network/wagmi/wagmi-chain-testnet/ws
```
Comment (patrick-ogrady) [inline docs/build/tutorials/platform/subnets/wagmi.md:38]: You should just add this under `RPC URL` as `WS URL`

File: docs/build/tutorials/platform/subnets/setup-dfk-node.md (hunk 1 above)
Comment (yulin-dong) [reply docs/build/tutorials/platform/subnets/setup-dfk-node.md:110]: The URLs are displayed on 
https://subnets.avax.network/defi-kingdoms/dfk-chain/explorer 
https://subnets.avax.network/defi-kingdoms/dfk-chain-testnet/explorer.

We don't need to add in this doc.

File: docs/build/tutorials/platform/subnets/wagmi.md (hunk 2 above)
Comment (yulin-dong) [reply docs/build/tutorials/platform/subnets/wagmi.md]: Done

**Pull request merged**